src/cloudant/_common_util.py

```diff
@@ -234,6 +234,24 @@ def codify(code_or_str):
         return _Code(code_or_str)
     return code_or_str
 
+def get_docs(url, **kwargs):
+    """
+    Provides a helper to get a response with documents for GET/POST requests.
+    """
+    r_session = kwargs.pop('r_session', None)
+    encoder = kwargs.pop('encoder', None)
+
+    params = python_to_couch(kwargs)
+    keys_list = params.pop('keys', None)
+    resp = None
+    if keys_list:
+        keys = json.dumps({'keys': keys_list}, cls=encoder)
```

Comment:
Looking at this code now I see that we have a possibility for a problem down the road.  Right now you are sending the `params` to be processed by `python_to_couch` function and right now it skips `keys` and does not perform any translation or conversion to the `keys` key or its value.  So you can get away with doing this `json.dumps` statement here.  If at any point the `python_to_couch` code changes and begins to process `keys` then you may run into a situation where you are performing multiple `json.dumps` commands on `keys`.

I know that you moved the code as it was originally but I think that had this same flaw.  This is probably a good time to correct that flaw and ensure that you pop off `keys` from `params` and handle `keys` as you do here and then pass `params` to `python_to_couch`.